content/docs/gke/authentication.md

```diff
@@ -7,15 +7,89 @@ weight = 4
 
 ## In-cluster authentication
 
-When you [set up Kubeflow for GCP](/docs/gke/deploy), it will automatically 
+When you [set up Kubeflow for GCP](/docs/gke/deploy), it will automatically
 [provision three service accounts](https://www.kubeflow.org/docs/gke/deploy/deploy-cli/#gcp-service-accounts) with different
-privileges. In particular, the `${KFAPP}-user` service account is meant to grant your user services access to GCP. 
+privileges. In particular, the `${KFAPP}-user` service account is meant to grant your user services access to GCP.
 The credentials to this service account can be accessed within the cluster as a
 [Kubernetes secret](https://kubernetes.io/docs/concepts/configuration/secret/).
 
 The secret will have basic access to a limited set of GCP services by default, but more roles can be granted through the
 [GCP IAM console](https://console.cloud.google.com/iam-admin/).
 
+### Namespaces
```

Comment:
I don't know if Namespaces makes sense a a section title. I'd probably call it "Creating Secrets", and then add the information from the previous paragraph about how some secrets are added automatically, but you have to manually create them in new namespaces

Reply:
How about "Distributing Secrets"?